### PR TITLE
Mentor/Mentee Pairing

### PR DESCRIPTION
**Pull request by sunkist5691**
Please don't merge this into MAIN (this is just for the testing purpose)

### CODE REVIEW

File: src/components/pages/Mentor/MentorPairing.js [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+      name="mentees"
+      style={{ width: '100%' }}
+      placeholder="Select a mentee"
+      showSearch
+      optionFilterProp="children"
+      onChange={value => setMenteeId(value)}
+    >
+      {mentees.filter(availableMentee).map(eachMentee => (
+        <Option key={eachMentee.id} value={eachMentee.id}>{`${
+          eachMentee.availability.as_early_as
```
Comment (joshwhitwell) [inline src/components/pages/Mentor/MentorPairing.js]: This data field on mentees should be "general_availability"

File: src/state/actions/index.js [hunk 2]
```diff
@@ -141,21 +141,73 @@ export const editSchool = (id, data) => dispatch => {
...
+// MENTORS
 // -----------------------
 
-export const fetchMentees = () => dispatch => {
-  dispatch({ type: actionTypes.FETCH_MENTEE_START });
+export const fetchMentors = () => async dispatch => {
+  dispatch({ type: actionTypes.FETCH_MENTOR_START });
+  const mentors = await axiosWithAuth().get('/mentor');
+
+  console.log('DID FETCH MENTOR');
```
Comment (joshwhitwell) [inline src/state/actions/index.js:209]: The commented promise handlers should be deleted since they are now obsolete with the async await syntax.

**Review by joshwhitwell — CHANGES_REQUESTED** [review]
Great job! Really good work on the MentorPairing component. I noticed you left some commented out promise handlers in the action creators for fetchMentor and fetchMentee. These should be deleted since they are obsolete with the async await syntax.

File: src/components/pages/Mentor/MentorPairing.js (hunk 1 above)
Comment (sunkist5691) [reply src/components/pages/Mentor/MentorPairing.js]: nice catch

**Review by sunkist5691 — COMMENTED** [review]
changed request:
 "deleting comment of promise handling"
 "change general_availability"

**Review by joshwhitwell — APPROVED** [review]

**Review by RohithSachdeva — APPROVED** [review]
Good job on the mentor pairing

**Pull request merged**